modules/monitoring-setting-scrape-and-evaluation-intervals-limits-for-user-defined-projects.adoc

```diff
@@ -46,31 +52,23 @@ metadata:
 data:
   config.yaml: |
     prometheus:
-      enforcedSampleLimit: 50000 <1>
+      enforcedSampleLimit: 50000 # <1>
+      enforcedLabelLimit: 500 # <2>
+      enforcedLabelNameLengthLimit: 50 # <3>
+      enforcedLabelValueLengthLimit: 600 # <4>
+      scrapeInterval: 1m30s # <5>
+      evaluationInterval: 1m15s # <6>
 ----
 <1> A value is required if this parameter is specified. This `enforcedSampleLimit` example limits the number of samples that can be accepted per target scrape in user-defined projects to 50,000.
-
-. Add the `enforcedLabelLimit`, `enforcedLabelNameLengthLimit`, and `enforcedLabelValueLengthLimit` configurations to `data/config.yaml` to limit the number of scraped labels, the length of label names, and the length of label values in user-defined projects:
```

Comment:
I didn't understand why we are removing this example and mentioned about only scrape interval

Reply:
Hello @slashpai , I merged it with the first example, as that is usually how we would have it in the docs. So instead of two steps that would happen simultaneously anyway, we have just one example YAML, as follows:

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: user-workload-monitoring-config
  namespace: openshift-user-workload-monitoring
data:
  config.yaml: |
    prometheus:
      enforcedSampleLimit: 50000 
      enforcedLabelLimit: 500 
      enforcedLabelNameLengthLimit: 50 
      enforcedLabelValueLengthLimit: 600 
      scrapeInterval: 1m30s 
      evaluationInterval: 1m15s 
```
You can see that the sample and label configurations are in just one example yaml instead of two. I hope that makes the change clearer. Thank you!

Reply:
The descriptions are there still for all the configurations, here is the preview for reference: https://85651--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-the-monitoring-stack.html#setting-scrape-and-evaluation-intervals-limits-for-user-defined-projects_configuring-the-monitoring-stack

Reply:
May be we should use same wordings and add info about scrape Interval and evaluation interval. May be split into multiple lines?

```
. Add the `enforcedLabelLimit`, `enforcedLabelNameLengthLimit`, and `enforcedLabelValueLengthLimit` configurations to `data/config.yaml` to limit the number of scraped labels, the length of label names, and the length of label values in user-defined projects:
```